### DISCUSSION TITLE
powershell

### DISCUSSION THREAD

**Discussion by stloc** [parcel-bundler/parcel#5650] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/4832070/104566236-a2e02080-564d-11eb-8579-750e2f98bacf.png)

is Parcel enable on Windows ?

**Comment by DeMoorJasper** [comment, marked as answer]
This looks like a configuration issue, parcel defines the parcel cli command in the package.json which should get picked up by yarn and exposed as a command. Shouldn't really make a difference which OS you're using

**Reply by mischnic** [reply]
You probably haven't added the yarn global bin dir to the PATH environment variable: https://sung.codes/blog/2017/12/30/yarn-global-add-command-not-work-windows/

**Reply by stloc** [reply]
@mischnic thanks a lot for this post. I tested succesfully !
